### PR TITLE
Barbeque/patch 1

### PR DESCRIPTION
**Pull request by CaseyFaist**
Running tests with Hatchet credentials (so they complete).

Supersedes #903 after updates 🌮 🎉 💜 

This update version bumps the readme to be in line with the Heroku dev center articles.

### CODE REVIEW

**Comment by hone** [comment]
Instead of listing the versions directly, should we just link out to DevCenter so we don't have to maintain two lists?

**Comment by CaseyFaist** [comment]
Possibly - the readme here needs a total overhaul, but in the meantime I think it's okay to make sure the supported version numbers are up to date

File: README.md [hunk 1]
```diff
@@ -61,8 +61,9 @@ Specify a Python Runtime
 
 Supported runtime options include:
```
Comment (hone) [inline README.md]:
```suggestion
Please see the list of [supported runtimes on DevCenter](https://devcenter.heroku.com/articles/python-support#supported-runtimes).
```

File: README.md [hunk 2]
```diff
@@ -61,8 +61,9 @@ Specify a Python Runtime
...
 
-- `python-3.7.4`
-- `python-3.6.9`
+- `python-3.8.2`
+- `python-3.7.6`
+- `python-3.6.10`
```
Comment (hone) [inline README.md]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -61,8 +61,9 @@ Specify a Python Runtime
...
 - `python-2.7.17`
```
Comment (hone) [inline README.md]:
```suggestion
```

**Pull request merged**